src/Raven.Server/Documents/Handlers/Processors/Subscriptions/AbstractSubscriptionsHandlerProcessorForDropSubscriptionConnection.cs

```diff
@@ -8,12 +11,10 @@ internal abstract class AbstractSubscriptionsHandlerProcessorForDropSubscription
         where TOperationContext : JsonOperationContext
```

Comment:
does this handler require proxy? how things will work from user/studio perspective?

Reply:
implemented `AbstractHandlerProxyNoContentProcessor` , need to add `nodeTag=A` (A is the responsible node) to the url, if not passed it will run on current node